run

```diff
@@ -159,7 +189,26 @@ clean() {
 restore() {
   if did restore ; then return 0 ; fi
 
-  paket --silent restore
+  if [ "$USE" = dotnet ] ; then
+    local NEW_CHECKSUM
+    if has md5 ; then
+      SUM=md5
+    elif has md5sum ; then
+      SUM=md5sum
+    else
+      echo "Couldn't find command to compute MD5."
+      return 1
+    fi
+    NEW_CHECKSUM="$(find . -name 'project.json' -print0 | xargs -0 cat | $SUM)"
+    local OLD_CHECKSUM=""
+    if [ -f .dotnet ] ; then OLD_CHECKSUM="$(cat .dotnet)" ; fi
+    if [ "$NEW_CHECKSUM" != "$OLD_CHECKSUM" ] ; then
+      echo -n "$NEW_CHECKSUM" > .dotnet
+      dotnet restore --verbosity Warning
```

Comment:
@polytypic why this check? the `dotnet restore already check if the file is changed